bundles/framework/announcements/instance.js

```diff
@@ -18,8 +18,46 @@ Oskari.clazz.define('Oskari.framework.bundle.announcements.AnnouncementsBundleIn
     }, {
 
         afterStart: function () {
-        }
+            var me = this;
+            if (me.started) {
+                return;
+            }
+            me.started = true;
+            me.sandbox = Oskari.getSandbox();
+            me.sandbox.register(me);
```

Comment:
I'm pretty sure this is already done on DefaultExtension.start() and could be removed here. You should be able to reference sandbox with just this.getSandbox() at this point.